clustered/client/src/test/java/org/ehcache/clustered/client/internal/UnitTestConnectionService.java

```diff
@@ -240,6 +246,8 @@ public static PassthroughServer remove(URI uri) {
           LOGGER.error("Entity destroy failed: ", ex);
         } catch (EntityNotFoundException ex) {
           LOGGER.error("Entity destroy failed: ", ex);
+        } catch (PermanentEntityException e) {
+          LOGGER.error("Entity destroy failed (permanent???): ", e);
```

Comment:
The exception is called `e` while the others are called `ex`. The 3 question marks are to say it is weird to have a permanent destroy?